templates/quickstart-database-for-atlassian-services.yaml

```diff
@@ -12,6 +12,14 @@ Parameters:
       - 'PostgreSQL'
     ConstraintDescription: Must be 'Amazon Aurora PostgreSQL' or 'PostgreSQL'.
     Type: String
+  DBEngineVersion:
+    Default: 9
```

Comment:
In our supported platforms page we say postgres 9.6, 10, and 11 and I wonder if we should explicitly put `9.6` as the opion instead of `9`
![image](https://user-images.githubusercontent.com/21027663/83822733-3d21d280-a715-11ea-8aff-36fbc400ad35.png)

Reply:
Yeah I was thinking about that, but then I was thinking about versions `11` and `10`. When they are selected the version that is actually deployed by CFN is `11.6` and `10.11` (the version prior to the latest) similarly when `9.6` is selected what you get is `9.6.16`. I guess my argument here is that regardless of what we list as the option, when selected that is never actually going to be exactly what you get.

Reply:
For me, I think it's important because it aligns with our documentation as customers are sticklers for being on supported versions.

Reply:
@ddomingorh may be able to give a better answer and I would defer to him.

Reply:
Okay, so... Turns out CFN template mappings can only be alphanumeric, meaning the options presented to the user will have to be:

- 9
- 10
- 11